Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -50,8 +50,11 @@ def __init__(self):
                          btnLabels=[x[0] for x in self.inversion_options],
                          callback=self._invalidate)
 
-        gui.auto_commit(self.controlArea, self, "autocommit", "Apply",
-                        checkbox_label="Apply on any change")
+        box = gui.widgetBox(self.controlArea, True, orientation="vertical")
+        box = gui.auto_commit(self.controlArea, self, "autocommit", "Apply",
+                              checkbox_label="Apply on any change")
+        gui.separator(box, 20)
+        box.layout().addWidget(self.report_button)
```

Comment:
If you ask me, revert this chunk, and we should rationalize Report button placement in OWWidget. I argue it can be done nicely (for standard, single-column widgets). But thankfully, @janezd has the final call. :smile: